common/transport/amqp/src/amqp_cbs.ts

```diff
@@ -136,7 +135,12 @@ export class ClaimsBasedSecurityAgent {
                       /*Codes_SRS_NODE_AMQP_CBS_16_020: [All responses shall be accepted.]*/
                       this._receiverLink.accept(msg);
                       for (let i = 0; i < this._putToken.outstandingPutTokens.length; i++) {
-                        if (msg.properties.correlationId === this._putToken.outstandingPutTokens[i].correlationId) {
+                        //
+                        // Just as a reminder.  For cbs, the message id and the correlation id
+                        // always stayed as string values.  They never went on over the wire
+                        // as the amqp uuid type.
```

Comment:
nit: "reminder", "always", and "never" seem to refer to tribal knowledge of other things - please keep the comments descriptive of what the correct things are as of now (our code is versioned anyway) and why they are. time is a construct :)